### PR TITLE
Replace FB issue tagging mentionees with dotnet teams

### PR DESCRIPTION
**Pull request by eiriktsarpalis**
Updates FB issue tagging so that @dotnet teams are tagged instead of individual github handles, making it more consistent with issue tagging in dotnet/runtime.

### CODE REVIEW

File: .github/fabricbot.json [hunk 1]
```diff
@@ -418,7 +405,6 @@
             "area-System.Formats.Cbor"
           ],
           "mentionees": [
-            "bartonjs",
             "dotnet/area-system-formats-cbor"
```
Comment (carlossanlop) [inline .github/fabricbot.json:408]: nit: spacing

File: .github/fabricbot.json [hunk 2]
```diff
@@ -444,48 +429,47 @@
             "area-System.Memory"
           ],
           "mentionees": [
-            "GrabYourPitchForks",
-            "carlossanlop"
+            "dotnet/area-system-memory"
```
Comment (carlossanlop) [inline .github/fabricbot.json:432]: I think you should keep @GrabYourPitchforks explicitly indicated here. He recently asked to be added to the notifications, even though he's not an area owner.

File: .github/fabricbot.json [hunk 3]
```diff
@@ -12,106 +12,99 @@
             "area-System.Security"
           ],
           "mentionees": [
-            "bartonjs",
+            "dotnet/area-system-security",
             "vcsjones",
-            "krwq",
-            "GrabYourPitchForks"
+            "krwq"
```
Comment (carlossanlop) [inline .github/fabricbot.json:17]: Did you mea  to keep krwq?

**Review by carlossanlop — COMMENTED** [review]
Left a couple of comments, @eiriktsarpalis .

**Review by gewarren — APPROVED** [review]

File: .github/fabricbot.json (hunk 3 above)
Comment (jeffhandley) [reply .github/fabricbot.json:17]: @krwq has remained subscribed to those notifications, so it was intentional. However, to make this more straightforward, I went ahead and added them to the dotnet/area-system-security team, so we can remove them from here.

File: .github/fabricbot.json [hunk 4]
```diff
@@ -12,106 +12,99 @@
...
         },
         {
           "labels": [
             "area-System.Text.Encoding"
           ],
           "mentionees": [
-            "tarekgh",
-            "dotnet/area-system-text-encoding"
+            "dotnet/area-system-text-encoding",
+            "tarekgh"
```
[the remark this answers is not in the document]
Comment (jeffhandley) [reply .github/fabricbot.json:66]: Tarek can be removed here; they're a member of the team.

File: .github/fabricbot.json [hunk 5]
```diff
@@ -12,106 +12,99 @@
...
         },
         {
           "labels": [
             "area-System.Text.Encodings.Web"
           ],
           "mentionees": [
-            "tarekgh",
-            "dotnet/area-system-text-encodings-web"
+            "dotnet/area-system-text-encodings-web",
+            "tarekgh"
```
Comment (jeffhandley) [inline .github/fabricbot.json:75]: I've added Tarek to this team so we can remove his explicit inclusion.

File: .github/fabricbot.json (hunk 2 above)
Comment (jeffhandley) [reply .github/fabricbot.json:429]: I confirmed that @GrabYourPitchforks is part of the memory team in GitHub.

**Review by jeffhandley — APPROVED** [review]

File: .github/fabricbot.json (hunk 5 above)
Comment (eiriktsarpalis) [reply .github/fabricbot.json]: FWIW we might want to make similar changes in the dotnet/runtime FB configs

Comment (jeffhandley) [reply .github/fabricbot.json]: I'll spin up a PR over there.

Comment (jeffhandley) [reply .github/fabricbot.json]: Done: https://github.com/dotnet/runtime/pull/64199

**Pull request merged**